### ISSUE TITLE
Reactive Stop and Start onClick

### ISSUE THREAD

**Issue by OmarOdaini** [slorenzo/react-stopwatch#211]
how can you stop or start the stopwatch with a button? I think thats so essential for a stopwatch!

**Comment by saeta-eth** [comment]
Hi @OmarOdaini.

Have you tried using `autoStart` property to start/pause the stopwatch?

**Comment by OmarOdaini** [comment]
@slorenzo thanks for the fast response!! so I was thinking that it'll be helpful to have `stop` `start` and `reset` functions

**Comment by saeta-eth** [comment]
You're right! This behavior should be handled but I'm not sure if your proposal is the right approach.

I think start/pause behavior can be handled with one property that will work as a toggle.

Let me see if I can find time to take a look at this.

**Comment by OmarOdaini** [comment]
Thanks :), I prefer your stop watch more than others but I just need that extra contol
